lib/elixir/lib/file.ex

```diff
@@ -1352,7 +1352,7 @@ defmodule File do
 
     * `:delayed_write`, `:raw`, `:ram`, `:read_ahead`, `:sync`, `{:encoding, ...}`,
       `{:read_ahead, pos_integer}`, `{:delayed_write, non_neg_integer, non_neg_integer}` -
-      for more information about these options see `:file.open/2`.
+      for more information about these options see [`:file.open/2`](https://erlang.org/doc/man/file.html#open-2).
```

Comment:
You don't need to link functions, those are automatically linked. Only modules aren't because they are ambiguous.

```suggestion
      for more information about these options see `:file.open/2`.
```

Reply:
Hi @josevalim,

While I'm reading a doc on https://hexdocs.pm/elixir/File.html#open/2, I came across this and notice I cannot click to go to erlang doc. Hence, I've added a link.

I'm not sure it is intedned. 


![image](https://user-images.githubusercontent.com/2466052/108959008-5e14c400-76a6-11eb-84e4-bc0ac7d5f9cb.png)

Reply:
Thanks, I will investigate!

Reply:
Hi @josevalim,

Is it because of the recent changes?

I run make docs locally on master branch and it seems the link is working.

The one that I read on https://hexdocs.pm/elixir/File.html#open/2 is v.1.11.3.

Reply:
It may be related to the Erlang version being used to generate the docs!